src/js/controllers/inviteCorrespondentDevice.js

```diff
@@ -42,25 +44,8 @@ angular.module('copayApp.controllers').controller('inviteCorrespondentDeviceCont
     correspondentListService.startWaitingForPairing(function(pairingInfo){
         console.log("beginAddCorrespondent " + pairingInfo.pairing_secret);
         $scope.code = pairingInfo.device_pubkey + "@" + pairingInfo.hub + "#" + pairingInfo.pairing_secret;
-
-        function determineQRcodeVersionFromString( inputtext ) {
```

Comment:
what was the point of moving the function to the service?

Reply:
because the function is now used in both wallethome.js and invitecorrespondentDevice.js

Reply:
wallethome.js is not included in the diff